admin_manual/maintenance/code-signing.rst

```diff
@@ -0,0 +1,177 @@
+============
+Code Signing
+============
+
+.. sectionauthor:: Lukas Reschke <lukas@owncloud.com>
+
+ownCloud supports code signing for the core releases as well as dedicated applications. Code signing gives our users an
+additional layer of security by ensuring that nobody else than authorized persons can push updates for your applications.
+
+It furthermore ensures that all upgrades have been executed properly. This means that no additional files are left as well
```

Comment:
»no additional files are left and old files were properly replaced«

I recommend running any text through this handy [Hemingway editor](http://www.hemingwayapp.com/) ;)